calyx/backend/src/verilog.rs

```diff
@@ -197,6 +197,9 @@ impl LibraryHandlerTrait for HardFloatHandler {
         let source_path =
             current_dir.join("primitives/float/HardFloat-1/source/");
 
+        // Include `RISCV` folder to include `iNFromException`
+        let riscv_path = source_path.join("RISCV");
```

Comment:
What is this supposed to do?

Reply:
This includes `iNFromException` in `HardFloat_specialize`, which is required by `recFNToIN`, which is required by `fpToInt`; and I picked RISCV out of RISCV/ARM-VFPv2/8086-SSE

Reply:
I dont think we can hardcode this into the compiler. Is there a better way?

Reply:
Well the HardFloat library hardcodes their directory names to `RISCV/ARM-VFPv2/8086-SSE`, we then have to pick of out of those hardcoded names to the include path right?

Reply:
Yeah sure, so I was thinking each library handler is meant for a different library; and right now we only have HardFloat library so only one handler.
For Morty, we need to specify the include directory paths for it to search symbols and cross-dependency include files to parse and pickle the Verilog files content. So that's why I think it's necessary to hardcode the source path: `primitives/float/HardFloat-1/source/`, as well as other cross-dependency inclusions, including but not limited to the `riscv_path = source_path.join("RISCV")` we are having here.
Please let me know if my explanation doesn't make sense.

Reply:
Understood! I think this makes more sense to me now!